### DISCUSSION TITLE
pr 후 언제 머지를 할 수 있는 건지?

### DISCUSSION THREAD

**Discussion by limhyoseok** [DeveloperAcademy-POSTECH/toGather#68] [General]
"pr 후 몇 명이상 승인이 나면 머지를 하기"처럼 규칙이 있으면 좋을 것 같습니다.

**Comment by KimDaeSeong8721** [comment]
깃관련해서 최소 1명이상 승인이 나면, PR올린 사람이 직접 머지하는 걸로 언급이 있었던 걸로 알고 있었습니다!

**Reply by yeongwooCho** [reply]
최소 1명 이상으로 강제하지 않았는데, 강제하는 건 어떻게 생각하시나요?

**Comment by limhyoseok** [comment]
2022.06.16 결론
- 최소 한명이 approve 해주면 머지하는 것으로
- 가장 적합한 사람을 리뷰어로 지정
